src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsServiceCollectionExtensions.cs

```diff
@@ -217,7 +217,6 @@ private static void AddListenerDecorators(IServiceCollection services)
         {
             // Order is important for these platform decorator registrations!
             // They will be applied in this order after any user registered decorators.
-            services.TryAddEnumerable(ServiceDescriptor.Singleton<IListenerDecorator, SingletonListenerDecorator>());
             services.TryAddEnumerable(ServiceDescriptor.Singleton<IListenerDecorator, FunctionListenerDecorator>());
```

Comment:
The remaining built in decorator applies a final outer error handling listener. I think it makes sense for us to retain this as the final outer listener (i.e. it can't be overridden by custom decorators).